.github/ci_commit_pins/xla.txt

```diff
@@ -1 +1 @@
-2eb4a60ed14a38260b85b0c765161f0ce45be6d1
+f71c02d1f457d58371e013632efb016c01bd1866
```

Comment:
This is not intended, is it?

Reply:
This is intended, see https://github.com/pytorch/pytorch/pull/137602#discussion_r1803427860